src/pages/loginPage/components/form.ts

```diff
@@ -37,21 +39,26 @@ const generateShowPasswordCheckbox = (): HTMLDivElement => {
 
 const generateEmailField = (): HTMLDivElement => {
   const wrapper = createWrapper();
+  const errorText = createTextElement();
+  errorText.classList.add('form__error-text_email');
   const label = createLabelElement();
   label.setAttribute('for', 'email');
   label.textContent = '* Email ';
   const input = createInputElement();
```

Comment:
Optional. I think it is better to improve createInputElement function with arguments. For example you can provide object as argument and pass to it className, attribute, etc. Inside createInput element you can desctucture this object and check if className passed, then add it to created element. For example:
function createInputElement({className}) {
  const el = createElement('input');
  if (className) {
    el.classList.add(className)
}
...(other checks and manipulations with required input)
return el;
}